docs/production/events.md

```diff
@@ -1,4 +1,4 @@
-# Events
+# Events (updated May 7, 2022)
```

Comment:
I've been questioning the value of dates in these headers. Is the git history insufficient?

Does it imply that the topic/implementation on such date, or that the docs have been updated then?
Is it meant to encourage future devs to update outdated docs?

or to warn devs that docs may be out of date? (this is the most compelling feature here, but it's easy for later doc-updaters to forget to update these days, yielding misinformed "docs out-of-date interpretations," unless we mitigate that risk)

Reply:
Maybe "updated" isn't the right term. Maybe it's "correct as of May 7, 2022".